### PR TITLE
add note re: kernel options

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Add content about advanced options for kickstart.

# Target branches

Which product version does this PR apply to?

- [ ] Manager-4.0
- [ ] Manager-4.1
- [x] Manager-4.2
- [x] Uyuni

# Links

Fixes https://github.com/SUSE/spacewalk/issues/12915

### CODE REVIEW

File: modules/client-configuration/pages/autoinst-profiles.adoc [hunk 1]
```diff
@@ -54,15 +54,20 @@ This setting allows {productname} to automatically pick the latest distribution
...
 [NOTE]
 ====
 Changing the Virtualization Type usually requires changes to the profile bootloader and partition options.
 This can overwrite your customization.
 Verify new or changed settings before saving them, by navigating to the [guimenu]``Partitioning`` tab.
 ====
 
+You can change the details and settings of your autoinstallation profiles by navigating to menu:Systems[Autoinstallation > Profiles] and clicking on the name of the profile you want to edit.
+
+To adjust advanced {kickstart} configuration options, navigate to the menu:Autoinstallation Details[Advanced Options] subtab.
```
Comment (mcalmer) [inline modules/client-configuration/pages/autoinst-profiles.adoc:66]: This page was renamed to Advanced Autoinstallation Configuration. See the after screenshot in the PR

**Review by mcalmer — CHANGES_REQUESTED** [review]

File: modules/client-configuration/pages/autoinst-profiles.adoc [hunk 2]
```diff
@@ -54,15 +54,20 @@ This setting allows {productname} to automatically pick the latest distribution
...
-You can change the details and settings of your autoinstallation profiles by navigating to menu:Systems[Autoinstallation > Profiles] and clicking on the name of the profile you want to edit.
-
 [NOTE]
 ====
 Changing the Virtualization Type usually requires changes to the profile bootloader and partition options.
 This can overwrite your customization.
 Verify new or changed settings before saving them, by navigating to the [guimenu]``Partitioning`` tab.
 ====
 
+You can change the details and settings of your autoinstallation profiles by navigating to menu:Systems[Autoinstallation > Profiles] and clicking on the name of the profile you want to edit.
```
Comment (keichwa) [inline modules/client-configuration/pages/autoinst-profiles.adoc:64]:
```suggestion
You can change the details and settings of your autoinstallation profiles by navigating to menu:Systems[Autoinstallation > Profiles] and clicking the name of the profile you want to edit.
```

File: modules/client-configuration/pages/autoinst-profiles.adoc [hunk 3]
```diff
@@ -54,15 +54,20 @@ This setting allows {productname} to automatically pick the latest distribution
...
+
+To adjust advanced {kickstart} configuration options, navigate to the menu:Autoinstallation Details[Advanced Options] subtab.
+For traditional clients, use the various fields to set options for specialized {kickstart} scenarios.
+For Salt clients, provide the kernel options you require.
+For more information about these options, see https://documentation.suse.com/sles/html/SLES-all/book-sle-admin.html.
```
Comment (keichwa) [inline modules/client-configuration/pages/autoinst-profiles.adoc:69]: I'm not sure whether it is possible to provide a more detailed reference to the SLE Admin guide.

**Review by keichwa — APPROVED** [review]

**Review by jcayouette — APPROVED** [review]

File: modules/client-configuration/pages/autoinst-profiles.adoc (hunk 3 above)
Comment (Loquacity) [reply modules/client-configuration/pages/autoinst-profiles.adoc:69]: Yeah, I wanted to remove the version from the URL, so it always goes to the latest.

File: modules/client-configuration/pages/autoinst-profiles.adoc (hunk 1 above)
Comment (Loquacity) [reply modules/client-configuration/pages/autoinst-profiles.adoc]: Good spot :+1:

File: modules/client-configuration/pages/autoinst-profiles.adoc [hunk 4]
```diff
@@ -54,15 +54,20 @@ This setting allows {productname} to automatically pick the latest distribution
...
 [NOTE]
 ====
 Changing the Virtualization Type usually requires changes to the profile bootloader and partition options.
 This can overwrite your customization.
 Verify new or changed settings before saving them, by navigating to the [guimenu]``Partitioning`` tab.
 ====
 
+You can change the details and settings of your autoinstallation profiles by navigating to menu:Systems[Autoinstallation > Profiles] and clicking the name of the profile you want to edit.
+
+To adjust advanced {kickstart} configuration options, navigate to the menu:Autoinstallation Details[Advanced Autoinstallation Configuration] subtab.
```
Comment (mcalmer) [inline modules/client-configuration/pages/autoinst-profiles.adoc:66]: I was referring to the first ˋkickstartˋ macro. You get there via a button which is called ˋAdvanced Optionsˋ 
So I am not sure about your change.

**Review by mcalmer — CHANGES_REQUESTED** [review]

File: modules/client-configuration/pages/autoinst-profiles.adoc (hunk 4 above)
Comment (Loquacity) [reply modules/client-configuration/pages/autoinst-profiles.adoc:66]: Oh, has the navigation changed? Can you give me a screenshot that shows the menu and tabs? (Or update refhead?)

Comment (mcalmer) [reply modules/client-configuration/pages/autoinst-profiles.adoc:66]: Here is the start page of Provisioning

image

You can see the button `Advanced Configuration`. When you click on it, you come to this page:

image

Here the title of the content box has changed: It is now called "Advanced Autoinstallation Configuration". 
In the past it was named "Advanced Kickstart Configuration".
This page looks a bit different when the system is a traditional managed client. Than we have additional configuration options. But the title has also changed.

image

Comment (Loquacity) [reply modules/client-configuration/pages/autoinst-profiles.adoc:66]: Ah ha, I understand now, I was navigating to the page quite differently. Updated, please check :)

**Review by mcalmer — APPROVED** [review]
Looks good

**Review by keichwa — APPROVED** [review]

**Pull request merged**